packages/storybook/stories/va-alert-expandable.stories.tsx

```diff
@@ -20,7 +20,7 @@ export default {
 
 const defaultArgs = {
   'status': 'info',
-  'trigger': 'Alert Headline',
+  'trigger': 'Example Headline',
```

Comment:
This example text was confusing with screen readers because it suggests that "Alert" is being read twice (ie. Warning Alert Alert Headline). I've updated it in Storybook to make it clearer that this is the example headline text and in a real situation this text would be more specific and likely would not have "Alert" in it.